src/pages/AssetDetailsPage.vue

```diff
@@ -24,29 +25,22 @@ const { width } = useWindowSize();
 
 const hashType = computed(() => (width.value < HASH_BREAKPOINT ? 'medium' : 'full'));
 
-const assetName = computed(() => {
+const assetDefinitionId = computed(() => {
   const name = router.currentRoute.value.params['id'];
+  const rest = router.currentRoute.value.hash;
```

Comment:
You should find a way to put `#` in the route in some escaped way (urlencode). It should not be a part of `route.hash`.